### DISCUSSION TITLE
今天的抢票报告

### DISCUSSION THREAD

**Discussion by peigenr** [mikumifa/biliTickerBuy#262] [聊天专区]
早上大会员提前抢设置500ms间隔手动过验证码，提前设置好开始时间后，到12：00时依然显示等待中，重启软件耽误了点时间，之后全都为too many requests for url，没有抢到，18：00场次提前30秒手动开始抢票，400ms间隔，依旧全程too many requests for url，手机端则为点日期都没反应，到有反应（大概2分钟后）的时候普票已全售完，电脑挂机蹲到10分钟的时候停在了如图（这图只是示例 不是当时的时间）
<img width="724" alt="20240629184015" src="https://github.com/mikumifa/biliTickerBuy/assets/135523209/6923c6c3-89ba-48ea-a9d5-184b938ad59d">
我以为是创建订单了，但进入会员购并没有待付款，重启软件解决，并设置100ms间隔直到现在依旧没有抢到票
请问米娜还有什么经验可以分享一下吗

**Comment by Kizuna-AI0630** [comment]
我不知道为啥，我开着手机会员购但是没操作，验证码竟然从手机上面出来的

**Reply by peigenr** [reply]
会这样（

**Comment by Hanabiiiiii** [comment]
提前设置好开始时间后，到12：00时依然显示等待中：可能是你日期设错了时间设对了，或者就是脚本bug
我抢到了，验证是手动验证，提前一分钟开抢，结合评论区间隔300ms似乎是成功率最高，手机端基本上我是放弃了，一直再试一次，手机准备好支付宝微信扫一扫，因为这次bw是付完款才锁单的，脚本上验证码一出立马扫。以上

**Reply by mikumifa** [reply]
我用的是本地时间，所以可能本地时间不太对

**Comment by peigenr** [comment]
问题就是有时候运行着突然就没反应了，点停止也没有反应，得重启软件才行

**Reply by Qilely** [reply]
卡住是不是因为因为 powershell的快速编辑模式给“假死了”

**Reply by peigenr** [reply]
不是 就是正常运行exe时莫名其妙就卡住了，应该就是#301中提到的问题？
